### PR TITLE
Increase mutation_event.keyword size.

### PR DESCRIPTION
**Pull request by angelicaochoa**
# What? Why?
This change is to prevent data truncation exception from being thrown. 

Changes proposed in this pull request:
* varchar(255) selected to match size of `mutation_event.protein_change` size.
* Change is also propagated to `mutation_count_by_keyword` table.

Signed-off-by: Angelica Ochoa <aochoa4230@gmail.com>

# Notify reviewers
@n1zea144 @sheridancbio 
@cBioPortal/backend

### CODE REVIEW

**Comment by sheridancbio** [comment]
Rebase needed. Since this is a schema change, I recommend we get input (or give notice) to other institutes. Also, maybe we should consider doing this in a release branch rather than hotfix ... so it goes through normal beta-testing before getting merged to master.
(having said that, I have a hard time imagining what could go wrong with expanding the width of a database field .. so in this case maybe we can be more confident)

**Comment by angelicaochoa** [comment]
@ersinciftci I don't think so - the keyword is generated by "guessing" the oncomutation keyword by searching for specific strings in the protein change (see `MutationKeywordUtils.guessOncotatorMutationKeyword(...)` for reference)

**Comment by angelicaochoa** [comment]
@n1zea144 @ersinciftci @sheridancbio I have updated my branch, this PR should be ready to merge once the tests complete

**Comment by jjgao** [comment]
@angelicaochoa should we try to merge this or just keep it open?

**Comment by angelicaochoa** [comment]
@jjgao let’s merge pleas. I’ll update my PR

**Comment by jjgao** [comment]
@angelicaochoa please feel free to merge since it's approved.

**Comment by angelicaochoa** [comment]
@jjgao I don't have write access

File: db-scripts/src/main/resources/migration.sql [hunk 1]
```diff
@@ -453,4 +453,7 @@ UPDATE info SET DB_SCHEMA_VERSION="2.5.0";
 ALTER TABLE gistic_to_gene DROP FOREIGN KEY gistic_to_gene_ibfk_2;
 ALTER TABLE gistic_to_gene ADD CONSTRAINT `gistic_to_gene_ibfk_2` FOREIGN KEY (`GISTIC_ROI_ID`) REFERENCES `gistic` (`GISTIC_ROI_ID`) ON DELETE CASCADE;
 
-UPDATE info SET DB_SCHEMA_VERSION="2.6.0";
+##version: 2.6.1
+ALTER TABLE `mutation_event` MODIFY COLUMN `KEYWORD` VARCHAR(255);
+ALTER TABLE `mutation_count_by_keyword` MODIFY COLUMN `KEYWORD` VARCHAR(255);
```
Comment (pieterlukasse) [inline db-scripts/src/main/resources/migration.sql]: @angelicaochoa you forgot to make this same modification in `cgds.sql`

**Pull request merged**